pom.xml

```diff
@@ -91,7 +92,7 @@
     <dependency.snakeyaml.version>2.2</dependency.snakeyaml.version>
     <dependency.spring.version>6.1.8</dependency.spring.version>
     <dependency.testcontainers.version>1.19.8</dependency.testcontainers.version>
-    <dependency.zeebe.version>8.6.6</dependency.zeebe.version>
+    <dependency.zeebe.version>8.6.7-SNAPSHOT</dependency.zeebe.version>
```

Comment:
temporarily as is depends on this fix https://github.com/camunda/camunda/pull/26419